backend/app/main/views.py

```diff
@@ -50,3 +50,25 @@ def get_all_clients():
             m = Client(name=name, notes=notes, attachments=attachments)
             list_of_clients.append(m.serialize())
     return jsonify(list_of_clients)
+
+
+# get a client from Airtable 
+@main.route("/clients/<id>", methods=["GET"])
+def get_a_client(id):
+    response = requests.get( 
+        "https://api.airtable.com/v0/appw4RRMDig1g2PFI/Clients/{}".format(id),
+        headers={"Authorization": str(os.environ.get("API_KEY"))},
+    )
+    print(response.status_code)
+    if response.status_code == 200: 
+        response_json = response.json()
+        client = []
+        name = response_json["fields"].get("Name")
```

Comment:
Remember to update these if the columns in Airtable get changed!

Reply:
Yes makes sense, thanks!